include/cutlass/epilogue/collective/xe_epilogue.hpp

```diff
@@ -300,22 +309,34 @@ class CollectiveEpilogue<
             make_shape(_, Int<FragsM>{}, Int<FragsN>{}, L),
             make_stride(Int<get<0>(MmaAtomShape{})>{}, Int<get<1>(MmaAtomShape{})>{}, _1{}));
 
+    // Because Nvidia uses shared memory, they are not tied to using the same accumulator values
```

Comment:
```suggestion
    // Because Sm90 uses shared memory, they are not tied to using the same accumulator values
```
SYCL has shared/local memory for H100s too.